schema/api.graphql

```diff
@@ -452,8 +453,64 @@ type Query {
 
     allGenres: [Genre!] @bind(class: "\\MyRadio\\ServiceAPI\\MyRadio_Scheduler", method: "getGenres") @auth(constants: [])
     allCreditTypes: [CreditType!] @bind(class: "\\MyRadio\\ServiceAPI\\MyRadio_Scheduler", method: "getCreditTypes") @auth(constants: [])
+
+    findMemberByName(name: String!, limit: Int): [MemberSearchResult!] @bind(class: "\\MyRadio\\ServiceAPI\\MyRadio_User", method: "findByName") @auth(constants: ["AUTH_APPLYFORSHOW"]) # TODO not 100% sure this is the best constant
+    findShowByTitle(term: String!, limit: Int!): [FindShowByTitleResult!] @bind(class: "\\MyRadio\\ServiceAPI\\MyRadio_Scheduler", method: "findShowByTitle") @auth(constants: ["AUTH_APPLYFORSHOW"]) # TODO ditto
+}
+
+type Mutation {
+    createShow(input: CreateShowInput): Show @bind(class: "\\MyRadio\\ServiceAPI\\MyRadio_Show", method: "create", callingConvention: FirstArgInput)
```

Comment:
I'm not sure how I feel about the `FirstArgInput` hack - Relay's style guide recommends having all mutations take one argument called `input`, but I feel like getting that to work with the ol' PHP stack will require more fuckery than it's worth.

Reply:
On the other hand, compare

```gql
mutation MyMutation($input: UpdatePostInput!) {
  updatePost(input: $input) { ... }
}
```
and
```gql
mutation MyMutation($id: ID!, $newText: String, ...) {
  updatePost(id: $id, newText: $newText, ...) { ... }
}
```

Reply:
I'll probably end up adding another calling convention that "splats" the input type across the arguments, kinda like we already do with query args.

Reply:
Done with `InputAsArgs`.